uavcan/__init__.py

```diff
@@ -45,6 +46,18 @@ def monotonic_wrapper(*args, **kwargs):
 class UAVCANException(Exception):
     pass
 
+class CanBus(object):
```

Comment:
We seem to be leveraging a different naming convention where acronyms are all capitalized: `CANBus`, `CANBusFD`, `CANBus2`.